### PR TITLE
Add 'rel="me"' link to verify our Mastodon presence

### PR DESCRIPTION
**Pull request by raboof**
https://docs.joinmastodon.org/user/profile/#verification

### CODE REVIEW

**Comment by drupol** [comment]
Is there a way to add a Mastodon icon, just like the rest of the links?

File: layout.tt [hunk 1]
```diff
@@ -140,6 +140,7 @@ BLOCK navigationLink %]
                 href="https://www.youtube.com/channel/UC3vIimi9q4AT8EgxYp_dWIw">Youtube</a></li>
             <li class="social-icon -github"><a href="https://github.com/NixOS">GitHub</a></li>
           </ul>
+          <a style="display: none" rel="me" href="https://chaos.social/@nixos_org">Mastodon</a>
```
Comment (drupol) [inline layout.tt:143]:
```suggestion
          <li class="social-icon -mastodon"><a href="https://chaos.social/@nixos_org">Mastodon</a></li>
```

Comment (drupol) [reply layout.tt:143]: You see what I mean, something like that.

**Comment by raboof** [comment]
> Is there a way to add a Mastodon icon, just like the rest of the links?

We can of course do that, but that might be a change that warrants some deliberation/discussion.

In this PR I wanted to just add the `rel="me"` validation, which I expect to be uncontroversial, so could be a quick merge.

**Comment by drupol** [comment]
You're totally right. Let's do one thing at a time ;)

File: layout.tt (hunk 1 above)
Comment (drupol) [reply layout.tt:143]: Please discard my comment.

**Review by drupol — APPROVED** [review]
All good here.

This change doesn't change the UI and does not add anything visible.

**Review by davidak — APPROVED** [review]

**Pull request merged**